ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/LmdGhost.java

```diff
@@ -78,16 +79,18 @@ public static BeaconBlock lmd_ghost(
         }
       }
 
+      final UnsignedLong max = max_value;
+
```

Comment:
I know this looks stupid. Apparently java lambdas requires final values for their comparisons. That's why I had to do it.